composer.required.json

```diff
@@ -35,6 +35,11 @@
       "Acquia\\Blt\\Custom\\": "../../../blt/src/"
     }
   },
+  "config": {
+    "platform": {
+      "php": "7"
```

Comment:
Curious about this answer...? Also, if this is the general approach of BLT, can't we undo #1829 as well?